apps/financial-aid/api/src/app/modules/auth/auth.service.ts

```diff
@@ -24,6 +24,16 @@ export class AuthService {
     return await res.json()
   }
 
+  async checkUserHistory(nationalId: string): Promise<boolean> {
+    const res = await fetch(
+      `${environment.backend.url}/api/me/?nationalId=${nationalId}`,
+      {
+        headers: { authorization: `Bearer ${environment.auth.secretToken}` },
+      },
+    )
+    return await res.json()
+  }
```

Comment:
We should probably move this to another file, kind off feel like this is user.service.ts or something similar to that